exporter/datadogexporter/factory.go

```diff
@@ -225,6 +237,14 @@ func (f *factory) createDefaultConfig() component.Config {
 			HostnameSource: HostnameSourceConfigOrSystem,
 		},
 	}
+	if isLogsAgentExporterEnabled() {
+		// logs agent sets this address by default to "https://agent-http-intake.logs.datadoghq.com"
```

Comment:
Is it okay to use an Agent endpoint to send log records from an application that is not the Agent? Have we checked?

Reply:
Looked into this more and decided to use the same endpoint as before "https://http-intake.logs.datadoghq.com/"